ArchiSteamFarm/Core/Utilities.cs

```diff
@@ -319,6 +327,42 @@ public static class Utilities {
 			}
 		}
 
+		internal static bool IsWeakPassword(string password, ISet<string>? additionallyForbiddenPhrases = null) {
```

Comment:
User probably would like to know, how to fix their own password - consider returning some kind of message, if one of the forbidden phrases or 3 repeating/sequential chars are in the password and password is considered weak.

Reply:
I'll add `, out string? weaknessReason`; We need some new strings tho. Do you have any suggestions?

Reply:
`Your IPC password seems to be weak due to {0}. Consider choosing a stronger one for increased security.`
`using words which is the name of ASF or its component`
`using sequential characters`
`using repetitive characters`
`using login of the account`

Reply:
We don't handle login name of the account different from other context-specific words 😅 That will just be the same as forbidden words. Tho I'll make sure to actually include the list of forbidden words.

I somehow dislike the fact that those strings need to have insight about the strings they are inserted into... I think I'll go with something more like the string we already have and just add `Reason: {1}` at the end 🤔 But thanks for the effort nontheless! 👍

Reply:
If adding reason, don't forget those should be localizable strings. Everything that can't be `nameof()` something must have localizable string, and sometimes even things that can be part of `nameof`.

Reply:
> If adding reason, don't forget those should be localizable strings.

Check the commits 😉 I already did that.